frontend/src/api/accountApi.js

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+import { Constants } from "../config/constants";
+
+const api = axios.create({
+  baseURL: `${Constants.idpUrl}/account/`
+});
+
+const AccountApi = {
+  deleteAccount: (email, password) => api.post("delete", { email, password })
```

Comment:
Same comment [as here](https://github.com/code4romania/stam-acasa/pull/461#discussion_r446568523)

Reply:
@surdu I tried getting it from session but I failed because this delete endpoint is on Identity Server and not API. Unfortunately no more ideas from my side